hw_3/task_06_format_text_output.py

```diff
@@ -0,0 +1,19 @@
+"""
+Пользователь вводит строку текста. Вывести каждое слово с новой строки.
+✔ Строки нумеруются начиная с единицы.
+✔ Слова выводятся отсортированными согласно кодировке Unicode.
+✔ Текст выравнивается по правому краю так, чтобы у самого длинного
+слова был один пробел между ним и номером строки.
+"""
+
+user_text = input("Введите строку текста: ")
+# user_text = "Hi, hi, hi! What an amazing meet!"
+
+text = user_text.split(' ')
```

Comment:
Если хотите разделить стропу по пробелам, то лучше ничего не передавать методу `split`.
```suggestion
text = user_text.split()
```